library/ssl_tls.c

```diff
@@ -9177,8 +9177,9 @@ int mbedtls_ssl_conf_psk( mbedtls_ssl_config *conf,
     ssl_conf_remove_psk( conf );
 
     /* Check and set raw PSK */
-    if( psk == NULL || psk_len > MBEDTLS_PSK_MAX_LEN )
+    if( psk == NULL || psk_len == 0 || psk_len > MBEDTLS_PSK_MAX_LEN)
```

Comment:
Missing space after MBEDTLS_PSK_MAX_LEN. Please run the check-names.sh script.

Reply:
corrected